actioncable/lib/action_cable/channel/base.rb

```diff
@@ -202,17 +212,21 @@ def transmit(data, via: nil)
         end
 
         def defer_subscription_confirmation!
-          @defer_subscription_confirmation = true
+          @defer_subscription_confirmation_counter.increment
         end
 
         def defer_subscription_confirmation?
-          @defer_subscription_confirmation
+          @defer_subscription_confirmation_counter.value.positive?
```

Comment:
Yep, just noticed failing specs for 2.2.x.

Maybe it's better to avoid additional deps and use `> 0`?